### PR TITLE
feat: adds opening external links via features

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

**Review by vinnyjth — COMMENTED** [review]
What's the difference between `OPEN_EXTERNAL_URL` and `OPEN_URL`?

**Comment by redreceipt** [comment]
opens safari on the front end. for giving links.

File: packages/apollos-data-schema/src/index.js [hunk 1]
```diff
@@ -870,6 +870,7 @@ export const featuresSchema = gql`
     READ_CONTENT
     READ_EVENT
     OPEN_URL
+    OPEN_EXTERNAL_URL
```
Comment (vinnyjth) [inline packages/apollos-data-schema/src/index.js:873]:
```suggestion
    OPEN_URL_EXTERNALLY
```

Comment (vinnyjth) [reply packages/apollos-data-schema/src/index.js:873]: Sorry to nitpick :) want to make this language as clear as possible.

Comment (redreceipt) [reply packages/apollos-data-schema/src/index.js]: yep i like it

**Pull request merged**